### DISCUSSION TITLE
SIMBA Athena ODBC issue multiple opening and closing cmd prompts , when auto refreshing tokens

### DISCUSSION THREAD

**Discussion by northdesk** [Versent/saml2aws#1066] [General]
We have just had to move from a ADFS connection to SAML2AWS , our tokens are set to expire after 1 hour . After following the set up instructions everything is working okay , but when reports are refreshed , there are multiple instances of a cmd prompt screen loading and unloading. Is there a way to hide this , as it makes working very difficult with hundreds of these instances every hours
my config settings are
[profile my_profile]
region = us-east-1
credential_process = saml2aws login --skip-prompt --quiet --credential-process --role xxxxx --profile xxxx

**Comment by mapkon** [comment]
CC: @gliptak
